cmd/govim/diagnostics.go

```diff
@@ -52,9 +49,9 @@ func (v *vimstate) diagnostics() []types.Diagnostic {
 			buf = types.NewBuffer(-1, fn, byts, false)
 		}
 		// make fn relative for reporting purposes
-		fn, err := filepath.Rel(cwd, fn)
+		fn, err := filepath.Rel(v.workingDirectory, fn)
```

Comment:
Per previous comments, worth caching the relative filepath (and we can fix that being "wrong" later, per our Slack chat) but definitely worth making identity based on absolute filepath.

Reply:
Ack